code/script.js

```diff
@@ -1,10 +1,262 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
 
+const nameInput = document.getElementById('name-input')
+
+const inputWrapper = document.getElementById('input-wrapper')
+
+/*const FoodReply = document.getElementById('input-wrapper') ----> hier stimmt etwas nicht */
+
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const handleNameInput = (event) => {
+  event.preventDefault()
+  // Store the value in a variable so we can access it after we 
+	// clear it from the input
+  const name = nameInput.value
+  showMessage(name, 'user')
+  nameInput.value = ''
+
+  // After 1 second, show the next option, passing the name into it
+  setTimeout(() => showTipOptions(name), 1000)
+}
+
+
+const showTipOptions = (userName) => {
+  // Post a greeting to the user as the 'bot'
+  showMessage(`Hi ${userName}. <br>
+  <div class="gifhey">
+    <iframe src="https://giphy.com/embed/l0ErUkqRz6Jug7RAI" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/obama-jimmy-fallon-michelle-obama-l0ErUkqRz6Jug7RAI">via GIPHY</a></p>
+  </div>
```

Comment:
You don't need the `div`, and also you could get rid of the text if you'd like. In that case, give the iframe a class name to style it:
```
<iframe
  class="gif"
  src="https://giphy.com/embed/l0ErUkqRz6Jug7RAI"
  frameBorder="0" class="giphy-embed"
  allowFullScreen>
</iframe>
```